sass/directives/_carousels.scss

```diff
@@ -105,7 +105,7 @@
   font-size: $direction-arrow-size / 2;
   cursor: pointer;
   border: 1px solid $inactive-carousel-button-color;
-  padding: $base-padding / 2;
+  padding: 0.4rem 0.5rem;
```

Comment:
Please switch this to a variable from our style base. If you want a smaller padding variable you could also add a general one (like `$base-padding--small`)